lambdas/jwkset/src/main/java/uk/gov/di/ipv/cri/address/api/handler/EnvironmentService.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.di.ipv.cri.address.api.handler;
+
+import java.util.Optional;
+
+public class EnvironmentService {
+    public String getEnvironmentVariableOrThrow(String envVar) {
+        return Optional.ofNullable(System.getenv(envVar)).orElseThrow();
+    }
+}
```

Comment:
☝️ you can't set env vars in java easily, so testing env vars by using a wrapping service is practical